PlanPluginBridge/src/main/java/com/djrapitops/pluginbridge/plan/luckperms/LuckPermsData.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Licence is provided in the jar as license.yml also here:
+ * https://github.com/Rsl1122/Plan-PlayerAnalytics/blob/master/Plan/src/main/resources/license.yml
+ */
+package com.djrapitops.pluginbridge.plan.luckperms;
+
+import com.djrapitops.plan.data.element.AnalysisContainer;
+import com.djrapitops.plan.data.element.InspectContainer;
+import com.djrapitops.plan.data.element.TableContainer;
+import com.djrapitops.plan.data.plugin.ContainerSize;
+import com.djrapitops.plan.data.plugin.PluginData;
+import com.djrapitops.plan.utilities.html.icon.Color;
+import com.djrapitops.plan.utilities.html.icon.Family;
+import com.djrapitops.plan.utilities.html.icon.Icon;
+import com.djrapitops.plan.utilities.html.icon.Icons;
+import java.util.Collection;
+import java.util.List;
+import java.util.OptionalInt;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import me.lucko.luckperms.api.*;
+import me.lucko.luckperms.api.caching.MetaData;
+
+public class LuckPermsData extends PluginData {
+    private LuckPermsApi api;
+
+    public LuckPermsData(LuckPermsApi api) {
+        super(ContainerSize.THIRD, "LuckPerms");
+        setPluginIcon(Icon.called("exclamation-triangle").of(Family.SOLID).build());
+
+        this.api = api;
+    }
+
+    @Override
+    public InspectContainer getPlayerData(UUID uuid, InspectContainer inspectContainer) {
+        User user = api.getUser(uuid);
+
+        if (user == null) {
+            inspectContainer.addValue("Data unavailable", "Could not get user data");
+            return inspectContainer;
+        }
+
+        MetaData metaData = user.getCachedData().getMetaData(Contexts.allowAll());
+        String prefix = metaData.getPrefix();
+        String suffix = metaData.getSuffix();
+
+        inspectContainer.addValue(getWithIcon("Primary group", Icon.called("user-friends").of(Family.SOLID)), user.getPrimaryGroup());
+
+
+        inspectContainer.addValue(getWithIcon("Prefix", Icon.called("file-signature").of(Family.SOLID).of(Color.GREEN)), prefix != null ? prefix : "None");
+        inspectContainer.addValue(getWithIcon("Suffix", Icon.called("file-signature").of(Family.SOLID).of(Color.BLUE)),suffix != null ? suffix : "None");
+
+        TableContainer metaTable = new TableContainer(
+                getWithIcon("Meta", Icon.called("info-circle").of(Family.SOLID)),
+                getWithIcon("Value", Icon.called("file-alt").of(Family.SOLID))
+        );
+        metaData.getMeta().forEach((key, value) -> metaTable.addRow(key, value));
+        inspectContainer.addTable("Meta", metaTable);
+
+        List<String> groups = user.getPermissions().stream().filter(Node::isGroupNode).map(Node::getGroupName).collect(Collectors.toList());
```

Comment:
I think a sorting call would be good, unless the groups are already in some nice order (Inheritance / Alphabetic)